apps/web/src/components/providers/SegmentProvider.tsx

```diff
@@ -9,7 +10,8 @@ type Props = {
 const SegmentContext = React.createContext<SegmentService>(undefined as any);
 
 export const SegmentProvider = ({ children }: Props) => {
-  const segment = React.useMemo(() => new SegmentService(), []);
+  const { mutate: track } = useTelemetry();
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Can we modify `useTelemetry` so we can do `const { track } = useTelemetry();` instead? This is a common pattern and object destructure renaming will be cumbersome + bad inner-DX‏